libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -17,6 +17,7 @@ import { Book } from '@tmo/shared/models';
 })
 export class BookSearchComponent implements OnInit {
   books: ReadingListBook[];
```

Comment:
This change looks great. You can remove the `books` member as well since it's not being used anymore.